aimsprop/fms90.py

```diff
@@ -320,27 +323,50 @@ def _create_frames_saddle(
                     q *= 2.0  # Upper/lower triangle
                 if abs(q) < cutoff_saddle:
                     continue  # Vanishing weight
-                frame = traj.Frame(
+                frame = bundle.Frame(
                     label=(I, J),
                     t=t,
                     w=q,
                     I=states[I],
                     N=Ns[I],
                     xyz=0.5 * (xyzs[I] + xyzs[J]),  # centroid
+                    widths=widths,
                 )
                 frames.append(frame)
 
     return frames
 
 
+def _parse_widths(filepath) -> List[float]:
+    """
+    parse the FMS.out tio get widths
+    """
+    fms_out = filepath / "FMS.out"
+
+    if fms_out.exists():
+        print(f"reading {fms_out}")
+        widths = []
+        with open(fms_out, "r") as f:
+            for line in f:
+                if re.search("Width:", line):
+                    a = float(line.split()[1])
+                    widths.append(a)
+
+    else:
+        print(f"{fms_out} does not exist.")
+        # TODO AV make a lookup table of usual widths
+        raise FileNotFoundError()
+    return widths
+
+
 def parse_fms90(
     filepath: Path,
     scheme: str = "mulliken",
```

Comment:
change this so there is no default value and manage error if they do not give a scheme